plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/HaHook.java

```diff
@@ -0,0 +1,39 @@
+package org.jenkins.tools.test.hook;
+
+import hudson.model.UpdateSite;
+import org.jenkins.tools.test.model.PomData;
+
+import java.util.Map;
+
+public class HaHook extends AbstractMultiParentHook {
```

Comment:
I am having the following error when trying to checkout this plugin:

```
Preparing for Multimodule checkout
Checking out from SCM connection URL: scm:git:git://github.com/cloudbees/jenkins-ha.git (jenkins-ha-parent-4.12)
[INFO] Executing: /bin/sh -c cd '/tmp/work' && 'git' 'clone' 'git://github.com/cloudbees/jenkins-ha.git' '/tmp/work/jenkins-ha'
[INFO] Working directory: /tmp/work
``` 
Seems like the checkout URL is malformed